pkgs/applications/video/olive-editor/default.nix

```diff
@@ -0,0 +1,41 @@
+{ stdenv, fetchFromGitHub, pkgconfig, which, qt5, frei0r, opencolorio, ffmpeg-full  }:
+
+stdenv.mkDerivation
+{
+  pname = "olive-editor";
+
+  version = "2019-03-18";
+
+  src = fetchFromGitHub {
+    owner = "olive-editor";
+    repo = "olive";
+    rev = "d153e4d7471122987098087a203323003ae8a128";
+    sha256 = "1gxpz6jg6dqjmvqkk0m9g4pz30lklwx51d73gka404a3w1j5wna6";
+  };
+
+  nativeBuildInputs = [ 
+    pkgconfig 
+    which 
+    qt5.qmake
+  ];
+
+  buildInputs = [
+    frei0r
+    ffmpeg-full
+    opencolorio
+  ] ++
+  (with qt5; [
```

Comment:
When you use `libsForQt5.callPackage` `qtbase` etc will be in scope, so you won't need the `with`.